.github/workflows/lint.yaml

```diff
@@ -18,15 +18,19 @@ on:
       - "synchronize"
       - "reopened"
 
-
 jobs:
   lint:
     name: "Lint & Publish Draft/Branch"
     runs-on: "ubuntu-latest"
     steps:
       - uses: "actions/checkout@v4"
       - uses: "authzed/actions/yaml-lint@main"
-      - uses: "bufbuild/buf-action@v1"
+      - name: "Lint Buf"
+        uses: "bufbuild/buf-action@v1"
         with:
           token: "${{ secrets.BUF_REGISTRY_TOKEN }}"
           breaking_against: "https://github.com/authzed/api.git#branch=main"
+      - name: "Lint Swagger docs"
```

Comment:
Why linting when the Swagger file is generated? What can we do if a failure arises?

Reply:
Good question. I double-checked the validation that it does and it doesn't seem very valuable, so I'll remove it. https://github.com/lirantal/swagger-lint-api?tab=readme-ov-file#usage